app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
```

Comment:
- [OPTIONAL ] I highly recommend making the wireframe for login the same as the given wireframe since in the given one the width for all the inputs is the same and it should be the same so please fix this as well in both the login form and signup form that would be amazing 👏🏻🔥💪🏻♻ I assign it as optional because the previous reviewer didn't mention it but it still important to add it 🙏🏻

- [ ] I highly recommend fixing the routes since when I open the app I should see signup form so the user can start entering his information but I should add it in the URL users/sign_up so please add them to your routes ♻🔥🙏🏻


- Your design
![image](https://github.com/yusufyusie/blog_app/assets/58553711/734cb217-f4ed-41cb-b4fe-f9c17f6e51b9)

- Given
![image](https://github.com/Ochiengsteven/art-post/assets/58553711/ed031123-04cd-4f2f-b5e5-6882f6133fe0)